ppmi_downloader/ppmi_downloader.py

```diff
@@ -157,7 +158,7 @@ def download_metadata(self, file_ids,
         self.driver.get('https://ida.loni.usc.edu/home/projectPage.jsp?project=PPMI')
         self.html.click_button("//a[text()='Download']")
         self.html.click_button("//a[text()='Study Data']")
-        self.html.click_button('//*[@id="ygtvlabelel56"]')
+        self.html.click_button('//*[@id="ygtvlabelel71"]')
```

Comment:
@mohanadarafe I had to change this because this button's id had changed. I feel like relying on button's labels rather than ids may be more reliable, do you think it's a good idea?

Reply:
sounds good!